how_to_build.md

```diff
@@ -19,27 +19,33 @@ The default [build profile](#building-only-a-subset-of-the-supported-target-fram
 
 Ensure that the following are installed:
 
-1. a recent version of Visual Studio 2019 (currently this means 16.8 or later) or the Build Tools for Visual Studio 2019
+1. The .NET Core 2.1 and 3.1 runtimes
 
-2. The .NET Core 2.1 and 3.1 runtimes
+2. The .NET 5.0 runtime
 
 3. The .NET Framework 4.6.1 or higher
 
-4. The targeting pack for .NET Framework 4.5
-
-5. An up-to-date version of the .NET 5.0 SDK (currently this means 5.0.100 or later)
+4. An up-to-date version of the .NET 6.0 SDK (currently this means 6.0.100 or later)
 
 You might not need everything to run a [partial build](#building-only-a-subset-of-the-supported-target-frameworks).
 
+In order to build from Visual Studio, you will also need:
+
+1. Visual Studio 2022 (17.0 or later)
+2. The .NET Framework 4.5 targeting pack. Unfortunately, this targeting pack is no longer available for download from Microsoft. As a workaround, you can get the necessary files from the `Microsoft.NETFramework.ReferenceAssemblies.net45` package:
+   1. Download the [Microsoft.NETFramework.ReferenceAssemblies.net45](https://www.nuget.org/packages/Microsoft.NETFramework.ReferenceAssemblies.net45/) package from NuGet
+   2. Open it as a ZIP
+   3. Extract the contents of `build\.NETFramework\v4.5` to the `C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.5` directory
+
 ### On Linux
 
-The default [build profile](#building-only-a-subset-of-the-supported-target-frameworks) on Linux builds FakeItEasy for .NET Standard 2.0, .NET Standard 2.1 and .NET 5.0, and runs the tests on .NET Core 2.1, .NET Core 3.1 and .NET 5.0 (the .NET Framework isn't supported on Linux).
+The default [build profile](#building-only-a-subset-of-the-supported-target-frameworks) on Linux builds FakeItEasy for .NET Standard 2.0, .NET Standard 2.1 and .NET 6.0, and runs the tests on .NET Core 2.1, .NET Core 3.1 and .NET 5.0 (the .NET Framework isn't supported on Linux).

```

Comment:
I think it's [still .NET 5.0 not .NET 6.0](https://github.com/thomaslevesque/FakeItEasy/blob/guardagainstnull-callerargumentexpression/profiles/full.props#L14)

Reply:
Oops! I replaced one too many

Reply:
Fixed
Also fixed the Linux requirements